packages/@webex/plugin-cc/src/AgentConfigService/types.ts

```diff
@@ -0,0 +1,232 @@
+/**
+ * Represents the request to UserRequest.
+ *
+ * @public
+ */
+
+export interface UserRequest {
+  /**
+   * Org id of the User
+   */
+  orgId: string;
+
+  /**
+   * CI user id of the User
+   */
+  ciUserId: string;
+}
+
+/**
+ * Represents the response from UserResponse.
+ *
+ * @public
+ */
+
+export interface UserResponse {
+  /**
+   * Identifier for a Desktop Profile.
+   */
+  agentProfileId: string;
+
+  /**
+   * This Specify the teams id which got assigned to the user.
+   */
+  teamIds: string[];
+
+  /**
+   * Identifier for an user profile which a Contact Center administrator has configured.
+   */
+  userProfileId: string;
+}
+
+/**
+ * Represents the request to DesktopProfileRequest.
+ *
+ * @public
+ */
+
+export interface DesktopProfileRequest {
```

Comment:
Where are we using this?

Reply:
I was using it before. We can remove this interface. Code Updated!